### ISSUE TITLE
Make PermCheck bin testable

### ISSUE THREAD

**Issue by renatomefidf** [eXistenZNL/PermCheck#3]
A good idea is to separate the command arguments and related classes to its own place, so it can also be unit tested.

**Comment by eXistenZNL** [comment]
Yeah, the whole bin/permcheck and src/PermCheck are a bit of a mess I guess. Let's discuss how I could improve this when I'm in the office again :)
